Readme.md

```diff
@@ -21,6 +21,7 @@ Every good Rocket Surgeon needs multiple choices of build systems to pick from t
 | Package | NuGet | MyGet |
 | ------- | ----- | ----- |
 | Rocket.Surgery.Nuke | [![nuget-version-6plqb7nwtdoa-badge]![nuget-downloads-6plqb7nwtdoa-badge]][nuget-6plqb7nwtdoa] | [![myget-version-6plqb7nwtdoa-badge]![myget-downloads-6plqb7nwtdoa-badge]][myget-6plqb7nwtdoa] |
+| Rocket.Surgery.Nuke.DotNetCore | [![nuget-version-75gp65y/nhyw-badge]![nuget-downloads-75gp65y/nhyw-badge]][nuget-75gp65y/nhyw] | [![myget-version-75gp65y/nhyw-badge]![myget-downloads-75gp65y/nhyw-badge]][myget-75gp65y/nhyw] |
```

Comment:
it worked!

Reply:
Azure DevOps is failing trying to push files that don't exist.  Go Go gadet yaml